### DISCUSSION TITLE
unsupported key: start when I try to access a playlist

### DISCUSSION THREAD

**Discussion by ttx99** [pystardust/ytfzf#635] [Q&A] [answered]
![Screenshot_2023-01-29_23-42-12](https://user-images.githubusercontent.com/95531944/215337750-3af310b0-52c1-4e49-b52c-6f3f9dcacbc5.png)

**Comment by Euro20179** [comment, marked as answer]
what version of fzf do you have

**Reply by ttx99** [reply]
updated to the latest fzf version 0.37.0 (2023012) solved it thanks.
